### ISSUE TITLE
Typo in `Formbidden` error message

### ISSUE THREAD

**Issue by mbiagetti** [EnterpriseDB/terraform-provider-biganimal#76]
Just a typo in case of 403 error from the provider here:

https://github.com/EnterpriseDB/terraform-provider-biganimal/blob/21e356db82b9539b75b899e80574a7647ed51376/pkg/api/errors.go#L8

### Expected Behavior
What should have happened?

### Actual Behavior

```
  Changes to Outputs:
    + continent = (known after apply)
    + status    = "ACTIVE"
  biganimal_region.this: Creating...

  Error: Formbidden

    with biganimal_region.this,
    on regions.tf line 2, in resource "biganimal_region" "this":
     2: resource "biganimal_region" "this" {
```
### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by serdardalgic** [comment]
OH yeah, good find, I was fixing it in the PR https://github.com/EnterpriseDB/terraform-provider-biganimal/pull/74/commits 😅

**Comment by serdardalgic** [comment]
This issue is fixed, as #74 is merged.
